release_creation/snapshot/freeze.sh

```diff
@@ -1,2 +1,7 @@
 set -e
-pip freeze --path ./target/$2 > $1
\ No newline at end of file
+pip freeze --path ./target/$2 > $1
```

Comment:
Name these out:
```suggestion
pip freeze --path "./target/$2" > "$1"
```

Also, quote your arguments to avoid weird globbing issues